source/__tests__/integration/create-property.test.js

```diff
@@ -276,14 +273,17 @@ describe('choose propertyTemplate from menu', () => {
 })
 
 async function expect_sel_to_exist(sel) {
+  await page.waitForSelector(sel)
   const sel_text = !!(await page.$(sel))
-  expect(sel_text).toEqual(true)
+  return expect(sel_text).toEqual(true)
```

Comment:
👍, i like returning the expectation result, even if it doesn't/didn't make a difference wrt async behavior.